samples/zigbee/light_switch/README.rst

```diff
@@ -59,9 +59,9 @@ Requirements
 
 The sample supports the following development kits:
 
-.. include:: /includes/boardname_tables/sample_boardnames.txt
-   :start-after: set8_start
-   :end-before: set8_end
+.. table-from-rows:: /includes/sample_board_rows.txt
+   :header: heading
+   :rows: nrf52840dk_nrf52840, nrf52833dk_nrf52833
```

Comment:
It's correct. 
However using overlay-fota.conf for nrf52833dk_nrf52833 is not possible because the sample becomes too big. I think we can inform about it in `Activating sample extensions` section.

Reply:
Could this be done in a separate PR?
This PR only changes the way the information is included, but the content is exactly the same as before. If the content needs to change, this isn't really related to this PR.

Reply:
@ru-fu ok

Reply:
Yes, I shall create another PR for adding this note.